Libraries/Text/TextProps.js

```diff
@@ -44,6 +44,8 @@ export type TextProps = $ReadOnly<{|
   accessibilityRole?: ?AccessibilityRole,
   accessibilityState?: ?AccessibilityState,
 
+  accessibilityLiveRegion?: ?('none' | 'polite' | 'assertive'),
```

Comment:
Why did this need to be added? Is `accessibilityLiveRegion` supported in `Text`? I don't see it listed in the [docs](https://reactnative.dev/docs/text).

This might be a case where the underlying example does not work. If that's the case, we could likely wrap the `Text` node in a `View` and pass `accessibilityLiveRegion` there instead.

Reply:
Earlier I was getting `Props` error and was saying `accessibilityLiveRegion` is not available in `TextProps`, that's why included.

**[Reference]** Can you please check this @kacieb  : [accessibilityliveregion-android](https://reactnative.dev/docs/accessibility#accessibilityliveregion-android) (it says `accessibilityLiveRegion` is allowed in `<Text>` as props.

You mean to say like this would be enough:
```suggestion
  <View accessibilityLiveRegion={'polite'}>
            <Text>Clicked {this.state.count} times</Text>
          </View>
```

If I add this code in `AccessibilityAndroidExample.android.js`.

Reply:
Oh thanks for the link, let me follow up with my team to see if this prop is supported in Text. If it is, this is quite the bug that it's not in the Flow type! If not, the docs are wrong. Thanks for pointing this out, I'll try to get back to you tomorrow on this!

Reply:
It looks like this is supported in Text - the Flow type was not up to date! So this seems fine to me, thanks for finding this!

Reply:
Hello! It looks like this is partially supported in Text at the moment, but not fully. Let's use your suggested changes (wrapping in a View) for now, and delete the addition to TextProps. I'll follow up on my end to get this fully implemented for Text.